lv_conf_template.h

```diff
@@ -603,6 +603,9 @@
 /** Enable drawing placeholders when glyph dsc is not found. */
 #define LV_USE_FONT_PLACEHOLDER 1
 
+/*Path to font file which is used in benchmark test*/
+#define LV_TEST_FONT_PATH "./lvgl/tests/src/test_files/fonts/noto/NotoSansSC-Regular.ttf"
```

Comment:
`lv_conf_template` needs to be synchronized with `Kconfig`, `LV_DEMO_BENCHMARK_FREETYPE_FONT_PATH`
`LV_DEMO_BENCHMARK_TINY_TTF_FONT_PATH`